src/tasks/__tests__/processBulkUpload.int.test.ts

```diff
@@ -201,6 +185,109 @@ describe('processBulkUpload', () => {
       getMockJobHelpers(),
     );
     const updatedBulkUpload = await loadBulkUpload(bulkUpload.id);
+    const { rows: proposals } = await db.sql<Proposal>(
+      'proposals.selectWithPagination',
+      {
+        offset: 0,
+        limit: 100,
+        search: '',
+      },
+    );
+    const firstProposal = proposals.find(
+      (proposal) => proposal.externalId === '1',
+    );
+    const secondProposal = proposals.find(
+      (proposal) => proposal.externalId === '2',
+    );
```

Comment:
Might be worth extracting a small helper so you can do like

```ts
const [firstProposal, secondProposal] = getProposals();
```

or whatever name for that method.